<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="UnsortedArrLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Unsorted Array</string>
    </property>
   </widget>
   <widget class="QLabel" name="Chooseyoursearchmethod">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>120</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose your search method</string>
    </property>
   </widget>
   <widget class="QLabel" name="Inputdatasetsize">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>346</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Input Dataset Size</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NormalSearchpb">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Normal Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BinarySearchpb">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Binary Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GenerateDatasetpb">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Dataset</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="InputDatasetSizeLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="MergeSortpb">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>490</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Merge Sort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="STLSortpb">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>490</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>STL Sort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbfindit">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>290</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Find it!</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SearchValueLineEdit">
    <property name="geometry">
     <rect>
      <x>552</x>
      <y>250</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Usingxsearch">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>170</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Using x search...</string>
    </property>
   </widget>
   <widget class="QLabel" name="found">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>350</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="ArrayDisplay">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>256</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="SortTypeLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
